src/packages/src/xroad/common/center/usr/share/xroad/scripts/restore_db.sh

```diff
@@ -84,45 +80,11 @@ REVOKE CREATE ON SCHEMA public FROM PUBLIC;
 EOF
 fi
 
-if ! detect_bdr; then
-    # restore dump
-    { cat <<EOF
-BEGIN;
-DROP SCHEMA IF EXISTS "$SCHEMA" CASCADE;
-EOF
-    cat "$DUMP_FILE"
-    echo "COMMIT;"
-    } | psql_dbuser >/dev/null || abort "Restoring database failed."
-else
-    echo "BDR 1.0 detected. BDR 1.0 is deprecated and support will be removed in a future X-Road release."
-    { cat <<EOF
-REVOKE CONNECT ON DATABASE "$DATABASE" FROM "$USER";
-SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname='$DATABASE' and usename='$USER';
-SET ROLE "$USER";
+# restore dump
+{ cat <<EOF
 BEGIN;
 DROP SCHEMA IF EXISTS "$SCHEMA" CASCADE;
 EOF
-    # Change statements like
-    # CREATE SEQUENCE centerui.anchor_urls_id_seq
-    #   START WITH 1
-    #   ...
-    #   USING bdr;
-    # to
-    # CREATE SEQUENCE <name>
-    # USING bdr;
-    # since BDR does not support most of the parameters (makes restore to fail)
-    sed -r -e '/^CREATE SEQUENCE /{:a;/;$/!{;N;ba};P;iUSING bdr;' -e ';d}' "$DUMP_FILE"
-
-    cat <<EOF
-COMMIT;
--- wait for changes to propagate before updating sequences
-SELECT bdr.wait_slot_confirm_lsn(NULL, NULL);
-SELECT pg_sleep(5);
-BEGIN;
-SELECT "$SCHEMA".fix_sequence('$SCHEMA');
```

Comment:
It would be also good to remove the now unnecessary fix_sequence stored procedure from the database schema (the baseline schema should not create the procedure any more)

Reply:
Removed in https://github.com/nordic-institute/X-Road/pull/1017/commits/135a98f26dc5af31f450876a96250e08f7f0ddb7